packages/wasm/README.md

```diff
@@ -92,6 +92,20 @@ Configures what code is emitted to instantiate the Wasm (both inline and separat
 - `"browser"` omits emitting code that requires node.js builtin modules that may play havoc on downstream bundlers
 - `"node"` omits emitting code that requires `fetch`
 
+### `exclude`
```

Comment:
please place these in the readme in alphabetical order 🙏

Reply:
Of course no problem, but I noticed that the former options is not in alphabetical order either, did you mean that to re-order all the options?